lib/resty/kafka/broker.lua

```diff
@@ -35,6 +35,14 @@ function _M.send_receive(self, request)
         return nil, err, true
     end
 
+    if self.config.ssl then
+      local _, err = sock:sslhandshake(true, self.host, self.config.ssl_verify)
+      if err then
+        return nil, "failed to do SSL handshake with " ..
+              self.host .. ":" .. tostring(self.port) .. ": " .. err, true
```

Comment:
style: `self.host` should align up with `"failed`.

Reply:
fixed.